terra-node-dev/test/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
add description / is this private?

Reply:
This is for testing only and needs none of that.